pom.xml

```diff
@@ -162,4 +164,59 @@
       </pluginRepository>
     </pluginRepositories>
 
+    <profiles>
+        <profile>
+            <id>license</id>
+            <build>
+                <defaultGoal>license:format</defaultGoal>
+                <plugins>
+                    <plugin>
+                        <groupId>com.mycila</groupId>
+                        <artifactId>license-maven-plugin</artifactId>
+                        <version>${mycila-license.version}</version>
+                        <configuration>
+                            <header>header.txt</header>
+                            <excludes>
+                                <exclude>KEYS</exclude>
+                                <exclude>**/NOTICE</exclude>
+                                <exclude>**/LICENSE</exclude>
+                                <exclude>**/NOTICE.txt</exclude>
+                                <exclude>**/LICENSE.txt</exclude>
+                                <exclude>doap.rdf</exclude>
+                                <exclude>**/README</exclude>
+                                <exclude>**/*.adoc</exclude>
+                                <exclude>**/node_modules/**</exclude>
+                                <exclude>**/cacerts</exclude>
+                                <exclude>**/*.p12</exclude>
+                                <exclude>**/*.txt</exclude>
+                                <exclude>.mvn/**</exclude>
+                                <exclude>mvnw*</exclude>
+                                <exclude>**/META-INF/persistence*.xsd</exclude>
+                                <exclude>**/MyRoutesWithPackage.java</exclude>
```

Comment:
`<exclude>**/pom.xml.versionsBackup</exclude>` would be handy in case we'll use versions plugin directly or indirectly via release plugin.

Reply:
done